src/Store/reducers/allDataTableReducer.js

```diff
@@ -6,6 +6,8 @@ const defaultState = {
   reassignSiteModalData: {},
   unenrollOpen: false,
   unenrollRowData: {},
+  editProtocolsModalOpen: false,
+  editProtocolsModalData: null,
```

Comment:
could you make this default to an object or list depending on what it will end up being, so that it's obvious what kind of data it will hold?

Reply:
This problem requires a bit more explanation when you have some time